drivers/media/i2c/adv7180.c

```diff
@@ -728,7 +808,7 @@ static int adv7180_set_pad_format(struct v4l2_subdev *sd,
 			break;
 		/* fall through */
 	default:
-		format->format.field = V4L2_FIELD_ALTERNATE;
+		format->format.field = V4L2_FIELD_INTERLACED;
```

Comment:
This is a partial revert of https://github.com/raspberrypi/linux/commit/6457b6263f0f11a9bff4e03a0141820ae9c81460
Why?
The chip produces 2 independent frames rather than line by line interleaved.
https://www.kernel.org/doc/html/latest/userspace-api/media/v4l/field-order.html#c.v4l2_field